tests/test_contrib/test_scatter.py

```diff
@@ -30,9 +29,15 @@
 from yellowbrick.exceptions import ImageComparisonFailure
 
 try:
-    import pandas
+    import pandas as pd
 except ImportError:
-    pandas = None
+    pd = None
+
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
```

Comment:
Importing the mock library in this fashion is to preserve python 2 compatibility; if we decide to drop python 2 support, then we can simply import from unittest.